resources/idlharness.js

```diff
@@ -841,7 +794,9 @@ IdlArray.prototype.collapse_partials = function()
 //@{
 {
     const testedPartials = new Map();
-    this.partials.forEach(function(parsed_idl)
+    this.partials
+        .sort((p1, p2) => p1.name > p2.name ? 1 : -1)
```

Comment:
Unlike all the other added `sort()` bits in this PR, `this.partials` isn't a newly created object, and this line makes we wonder if it's used anywhere else before being sorted. Maybe sort it right after it's populated? Or just leave it in the order it is, since the order will depend on the IDL input and should be reliable?